mafia/data/garbo_settings.json

```diff
@@ -51,6 +51,11 @@
     "type": "boolean",
     "description": "**WARNING: Experimental** Don't show user confirm dialogs, instead automatically select yes/no in a way that will allow garbo to continue executing. Useful for scripting/headless. Risky and potentially destructive."
   },
+  {
+    "name": "garbo_autoUserConfirm_embezzlerInvocationsCount",
+    "defaultValue": 1,
+    "description": "This is used in conjunction with autoUserConfirm, will automatically use resources this many times to invoke an embezzler before requesting user interference."
```

Comment:
I think the description should be more explicit that it only functions if autoUserConfirm is enabled, and that invoking means wishing, using a clover, etc.

Reply:
Also, it should specify that it is up to that many times, not guaranteed that many times.